local/store.go

```diff
@@ -360,6 +360,25 @@ func (s *Store) SaveFile(image *Image, withName string) (string, error) {
 		return "", err
 	}
 
+	image.Image, err = imgutil.MutateManifest(image.Image, func(mfest *v1.Manifest) (mutateSubject, mutateAnnotations bool) {
```

Comment:
Even though a local.Image implements the v1.Image interface, the tarball generated from SaveFile is in the legacy docker format (not OCI layout) so I don't think this does anything.